applications/nrf5340_audio/src/audio/audio_sync_timer.c

```diff
@@ -79,6 +86,89 @@ static void event_handler(nrf_timer_event_t event_type, void *ctx)
 {
 }
 
+static void nrfx_gpiote_handler(void)
+{
+}
+
+static int audio_sync_timer_sync_led_init(void)
+{
+	int ret;
+
+	/* Connect the IRQ handler for timer 1 */
+	if (AUDIO_SYNC_TIMER_INSTANCE == 1) {
+		IRQ_CONNECT(DT_IRQN(DT_NODELABEL(timer1)), AUDIO_SYNC_TIMER_IRQ_PRIORITY, nrfx_isr,
+			    nrfx_timer_1_irq_handler, 0);
+	} else {
+		LOG_WRN("Invalid audio sync timer instance");
+		return -ECANCELED;
+	}
+
+	if (nrfx_gpiote_is_init()) {
+		LOG_DBG("GPIOTE is already initiated");
+	} else {
+		ret = nrfx_gpiote_init(NRFX_GPIOTE_DEFAULT_CONFIG_IRQ_PRIORITY);
+		if (ret - NRFX_ERROR_BASE_NUM) {
+			LOG_ERR("Failed to init GPIOTE (err: %d).", ret);
```

Comment:
Nitpick:

```suggestion
			LOG_ERR("nrfx GPIOTE init error - Return value: %d", ret);
```

We usually dont have periods in logs